ee/api/billing.py

```diff
@@ -126,7 +126,13 @@ def deactivate(self, request: Request, *args: Any, **kwargs: Any) -> HttpRespons
         if not product:
             raise ValidationError("Products must be specified")
 
-        BillingManager(license).deactivate_products(organization, product)
+        try:
+            BillingManager(license).deactivate_products(organization, product)
+        except Exception as e:
+            detail_object = e.args[2]
+            return Response(
+                {"statusText": e.args[0], "detail": detail_object["error_message"]}, status=status.HTTP_400_BAD_REQUEST
```

Comment:
Security hat on - are we certain there are no ways the billing service would responsd with info we wouldn't want leaked out?

Reply:
Yeah, it's a good point. I've tried to scope this change only to the deactivate product endpoint and a specific Exception type on that endpoint to avoid this issue. I'm still not entirely certain on this approach so am happy to take pointers on how this might be structured better.